pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -1430,8 +1433,8 @@ func (r *ReconcileHyperConverged) ensureIMSConfig(req *hcoRequest) *EnsureResult
 		return res.Error(errors.New("ims-conversion-container not specified"))
 	}
 
-	if os.Getenv("VMWARE_CONTAINER") == "" {
-		return res.Error(errors.New("ims-vmware-container not specified"))
+	if os.Getenv("IMPORT_PROVIDER_CONTAINER") == "" {
```

Comment:
how do we handle upgrades?
shouldn't we deleted the old config map? And transfer the vddk image from the old one if no VDDK_INIT_CONTAINER is found (since we expect users to produce it)?